packages/salesforcedx-vscode-apex/src/commands/apexActionController.ts

```diff
@@ -132,4 +130,127 @@ export class ApexActionController {
       .join('\n');
     return stringify(parse(theDoc));
   }
+
+  private saveOasAsErsMetadata = async (oasSpec: string, fullPath: string): Promise<void> => {
+    // Replace the schema section in the ESR file if it already exists
+    let existingContent;
+    if (fs.existsSync(fullPath)) {
+      existingContent = fs.readFileSync(fullPath, 'utf8');
+    }
+    const namedCredential = await this.showNamedCredentialsQuickPick();
```

Comment:
@CristiCanizales I see that `showNamedCredentialsQuickPick` throws an error if something goes sideways on the query. What happens if there is not a default org available? Should we issue a message and then ask the user to enter a named credential?